packages/mgt-chat/src/components/ManageChatMembers/ListChatMembers.tsx

```diff
@@ -21,7 +21,7 @@ interface ListChatMembersProps {
   closeParentPopover: () => void;
 }
 
-const RemovePerson = bundleIcon(Dismiss24Regular, () => null);
```

Comment:
This was so that the remove icon only showed on hover, which it now occurs to me that it also needs to show when the element has focus.... This needs some testing, but let's just leave it as is for the moment and not introduce secondary changes into this PR please